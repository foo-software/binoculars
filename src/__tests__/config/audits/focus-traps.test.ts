import auditFocusTraps from '../../../config/audits/focus-traps';

describe('focus-traps audit', () => {
  it('should match snapshot', () => {
    expect(auditFocusTraps('en')).toMatchSnapshot();
  });
});
