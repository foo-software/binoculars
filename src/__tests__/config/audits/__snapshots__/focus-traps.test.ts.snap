// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`focus-traps audit should match snapshot 1`] = `
Object {
  "description": "A user can tab into and out of any control or region without accidentally trapping their focus. [Learn more](https://web.dev/focus-traps/).",
  "title": "User focus is not accidentally trapped in a region",
}
`;
